src/test/java/edu/cornell/kfs/module/purap/document/service/impl/CuCreditMemoServiceImplTest.java

```diff
@@ -194,6 +182,10 @@ class TestCuCreditMemoServiceImpl extends CuCreditMemoServiceImpl {
         public VendorCreditMemoDocument getCreditMemoDocumentById(Integer purchasingDocumentIdentifier) {
             return setupVendorCreditMemoDocument();
         }
+        
+        protected void reIndexDocument(VendorCreditMemoDocument document) {
```

Comment:
Is this override needed if you're using the `Mockito.doXXX()` methods for stubbing? I'm also wondering if the enclosing `TestCuCreditMemoServiceImpl` class could be removed entirely.

Reply:
yes, TestCuCreditMemoServiceImpl can be removed